src/Model/APContact.php

```diff
@@ -272,6 +272,11 @@ public static function getByURL($url, $update = null)
 
 		if (!empty($apcontact['followers'])) {
 			$followers = ActivityPub::fetchContent($apcontact['followers']);
+			// Mastodon seriously allows for this condition? 
+			// Jul 14 2021 - See https://mastodon.online/@goes11 for a negative followers count
+			if ($followers['totalItems'] < 0) {
+			  $followers['totalItems'] = 0;
```

Comment:
Code standards: please use tabulations for indentation.
```suggestion
				$followers['totalItems'] = 0;
```

Reply:
Shoot, sorry about that!

Reply:
It's okay, that's why code review exists!